fh-android-sdk/src/main/java/com/feedhenry/sdk/FH.java

```diff
@@ -418,10 +424,33 @@ public static int getLogLevel() {
 
     /**
      * Get the customized user-agent string for the SDK
-     * 
+     *
      * @return customized user-agent string
      */
     public static String getUserAgent() {
         return Device.getUserAgent();
     }
+
+    /**
+     *
+     * Registers a device to a push network.
+     *
+     * @param pContext  your application's context
+     * @param pCallback the pCallback function to be executed after the device registration is finished
+     */
+    public void pushRegister(final Context pContext, final FHActCallback pCallback) {
+        RegistrarManager.config("fh", AeroGearGCMPushConfiguration.class)
```

Comment:
Does this load the push variants from a file? Probably worth adding it to the java docs to make it clear.